test/endpoints/automations.test.js

```diff
@@ -136,55 +276,70 @@ describe('Tests for automation endpoints\n', () => {
         expect(res.body)
           .to.have.property('message')
           .to.equal('Successfully fetched automations');
-        expect(res.body.pagination)
-          .to.have.property('currentPage')
-          .to.be.equal(1);
+        expect(res.body.data[0])
+          .to.have.property('type')
+          .to.be.equal('offboarding');
         done();
       });
   });
   it('Should search automation data for partners or fellows name containing a string', (done) => {
     chai
       .request(app)
-      .get('/api/v1/automations?searchTerm=conroy')
+      .get('/api/v1/automations')
       .end((err, res) => {
         expect(res).to.have.status(200);
         expect(res.body)
           .to.have.property('message')
           .to.equal('Successfully fetched automations');
-        expect(res.body.pagination)
-          .to.have.property('currentPage')
-          .to.be.equal(1);
-        done();
+        const { fellowName } = res.body.data[0];
+        chai
+          .request(app)
+          .get(`/api/v1/automations?searchTerm=${fellowName}`)
+          .end((error, response) => {
```

Comment:
check that the automation being return does actually have a fellow/engineer with the searchTerm in it